.github/CODEOWNERS

```
@@ -77,6 +77,12 @@
 # NixOS integration test driver
 /nixos/lib/test-driver  @tfc
 
+# Systemd
+/nixos/modules/boot/systemd.nix      @NixOS/systemd @Kloenk
+/nixos/modules/boot/systemd-*.nix    @NixOS/systemd @Kloenk
```

`/nixos/modules/boot` doesn't exist.

```suggestion
/nixos/modules/system/boot/systemd.nix      @NixOS/systemd @Kloenk
/nixos/modules/system/boot/systemd    @NixOS/systemd @Kloenk
```